app/src/main/AndroidManifest.xml

```diff
@@ -26,7 +27,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        tools:ignore="GoogleAppIndexingWarning">
+        android:requestLegacyExternalStorage="true"
```

Comment:
So this is required on the app level? It's breaking change for users, seems it's better to roll back 29 api level then

Reply:
Or, maybe - we can keep it here BUT not in mapbox-vision

Reply:
Actually it can be removed from app module, we do not access any truly external files there.

Reply:
Removed.